python/GangaLHCb/__init__.py

```diff
@@ -84,28 +65,20 @@ def _store_root_version():
 def _store_dirac_environment():
     from GangaDirac.Lib.Utilities.DiracUtilities import write_env_cache, get_env
     platform = os.environ['CMTOPT']
-    diracversion = Ganga.Utility.Config.getConfig('LHCb')['diracVersion']
+    diracversion = Ganga.Utility.Config.getConfig('LHCb')['DiracVersion']
     fdir = os.path.join(os.path.expanduser("~/.cache/Ganga/GangaLHCb"), platform)
-    fname = os.path.join(fdir, defaultLHCbDirac)
+    fname = os.path.join(fdir, diracversion)
     if not os.path.exists(fname) or not os.path.getsize(fname):
-        logger.info("Storing new LHCbDirac environment (%s:%s)" % (str(defaultLHCbDirac), str(platform)))
-        cmd = 'lb-run LHCBDIRAC {version} python -c "import os; print(dict(os.environ))"'.format(version=defaultLHCbDirac)
-        env = execute(cmd)
+      cmd =  'lb-run LHCBDIRAC {version} python -c "import os; print(dict(os.environ))"'.format(version=diracversion)
+      env = execute(cmd)
+      if len(env)!=0:
```

Comment:
Yes I see. Would it be preferable to put the check into execute and get that to return a failure or just have the check here? I think a check only here would be best in case other things don't mind if the executed command doesn't run perfectly.